var/spack/repos/builtin/packages/vapor/package.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+import re
+
+from spack.package import *
+
+
+class Vapor(CMakePackage):
+    """VAPOR is the Visualization and Analysis Platform for Ocean,
+    Atmosphere, and Solar Researchers. VAPOR provides an interactive 3D
+    visualization environment that can also produce animations and
+    still frame images.
+    """
+
+    homepage = "https://www.vapor.ucar.edu"
+    url = "https://github.com/NCAR/VAPOR/archive/refs/tags/v3.9.0.tar.gz"
+    git = "https://github.com/NCAR/VAPOR.git"
+
+    maintainers("vanderwb")
+
+    version("main", branch="main")
+    version(
+        "3.9.0",
+        sha256="343ababe40b5824ef826f16c935a6dc1fb18e1a4c88ef967c8d64386f28a99a3",
+        preferred=True,
+    )
+
+    variant("doc", default=True, description="Build docs using Doxygen")
+    variant("ospray", default=False, description="Enable OSPRay raytracing")
+
+    depends_on("cmake@3.17:", type="build")
+    depends_on("xz")
+    depends_on("zlib")
+    depends_on("openssl")
+    depends_on("expat")
+    depends_on("curl")
+    depends_on("which")
```

Comment:
Is it:
```suggestion
    depends_on("which", type="run")
```
?

Reply:
I added this dependency based on the developer's install instructions, but actually it isn't necessary for this Spack build and so I have removed it. Thanks for suggesting the second look. I've checked the other dependencies and I do believe all others are justified.